install.yaml

```diff
@@ -7,12 +7,16 @@ name: oci8
 # DDEV environment variables can be interpolated into these actions
 # pre_install_actions are executed in the context of the target project's root directory.
 pre_install_actions:
-  # - |
-  #   #ddev-description:Check architecture type for incompatible arm64 type
-  #   if [ "$(uname -m)" = "arm64" -o "$(uname -m)" = "aarch64" ]; then
-  #     echo "This package does not work on arm64 (Apple Silicon) machines";
-  #     exit 1;
-  #   fi
+  - |
+    #ddev-description:Check architecture type for incompatible arm64 type
+    arch="$(uname -m)"
+    os="$(uname -s)"
+
+    if [ "$os" != "Linux" ] || { [ "$arch" != "x86_64" ] && [ "$arch" != "aarch64" ]; }; then
+      echo "This package only works on Linux AMD64 or ARM64 processors."
+      exit 1
+    fi
+
```

Comment:
Sorry, this *does* now work with arm64. That's why the upstream was commented out.

```suggestion
```

Reply:
@rfay, I added those lines to check the current architecture, which is why I removed the Docker Compose file, as it only specifies platform: linux/amd64. But since you confirmed it doesn’t work, I’ve removed those lines as well.